src/components/Dropdown/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@further-ventures/fv-react-ui-components.dropdown",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "classnames": ">=2.0.0",
+    "react": ">=18.0.0"
+  }
```

Comment:
There are dependencies required by component like `Input`, `Button`, `react-transition-group` etc. Could you please add them as well?

Reply:
Aren't the dependencies to local components added automatically?

Reply:
Nope, the list of external dependencies is based on the components `package.json` files